src/containers/App/app.mod.scss

```diff
@@ -1,12 +1,11 @@
 @import "../../scss/vars.scss";
 
-$footer-bottom: 44px;
+$footer-bottom: 60px;
 
 .appContainer {
   background-color: $bg-color;
   height: calc(100vh - #{$footer-bottom});
-
-  padding-bottom: $footer-bottom;
+  overflow: scroll;
```

Comment:
Please make sure this is only scroll-y, otherwise it gets annoying whenever content overflows